src/main/java/dev/redstudio/alfheim/Alfheim.java

```diff
@@ -14,7 +14,7 @@
 //  | $$  | $$| $$| $$      | $$  | $$| $$_____/| $$| $$ | $$ | $$
 //  | $$  | $$| $$| $$      | $$  | $$|  $$$$$$$| $$| $$ | $$ | $$
 //  |__/  |__/|__/|__/      |__/  |__/ \_______/|__/|__/ |__/ |__/
-@Mod(modid = ID, version = VERSION, updateJSON = "https://forge.curseupdate.com/910715/" + ID, useMetadata = true, dependencies = "required-after:mixinbooter@[8.8,);required-after:redcore@[0.5,)")
+@Mod(modid = ID, version = VERSION, updateJSON = "https://forge.curseupdate.com/910715/" + ID, useMetadata = true, dependencies = "required-after:mixinbooter@[8.8,);required-after:redcore@[0.5,)", acceptableRemoteVersions = "*")
```

Comment:
In what scenario would you want mismatched versions? this could cause issues so it's better to have a real use case